src/mscorlib/src/System/String.cs

```diff
@@ -3778,6 +3778,16 @@ internal unsafe static void InternalCopy(String src, IntPtr dest,int len)
                 Buffer.Memcpy(dstPtr, srcPtr, len);
             }
         }      
+
+        // This is only intended to be used by char.ToString.
+        // It is necessary to put the code in this class instead of Char, since m_firstChar is a private member.
+        // Making m_firstChar internal would be dangerous since it would make it much easier to break String's immutability.
+        internal static string CreateFromChar(char c)
```

Comment:
I put it near the constructors, which are near the `CreateStringFromEncoding` / `CreateString` methods.